### PR TITLE
Improve the wording of README.md slightly

### PR DESCRIPTION
**Pull request by psmears**
Make a number of minor changes to fix typos etc

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -112,12 +112,12 @@ Output:
    Spec file golang-github-cpuguy83-go-md2man.spec at /tmp/test/golang-github-cpuguy83-go-md2man/fedora/golang-github-cpuguy83-go-md2man
    ```
 
-At the beginning, golang checks the Fedora repository if the package already exists.
-If not, it creates a spec file (needs to be filled for missing data),
+At the beginning, golang checks the Fedora repository to see if the package already exists.
+If not, it creates a spec file (needs to have issing data filled in),
```
Comment (ingvagabund) [inline README.md]: s/issing/missing

**Comment by ingvagabund** [comment]
@psmears great, thanks for the corrections

File: README.md (hunk 1 above)
Comment (psmears) [inline README.md]: Good spot! Fixed :)

**Comment by ingvagabund** [comment]
@psmears appreciated sir :)

**Pull request merged**